lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -143,21 +143,13 @@ def _add_bounds_to_percentiles_and_forecast_at_percentiles(
         constant lower and upper bounds.
         Args:
             percentiles (Numpy array):
-                Array of percentiles from a Cumulative Distribution Function.
-            forecast_at_percentiles (Numpy array):
-                Array containing the underlying forecast values at each
-                percentile.
-            bounds_pairing (Tuple):
-                Lower and upper bound to be used as the ends of the
-                cumulative distribution function.
-        Returns:
-            (tuple) : tuple containing:
-                **percentiles** (Numpy array):
-                    Array of percentiles from a Cumulative Distribution
-                    Function.
-                **forecast_at_percentiles** (Numpy array):
-                    Array containing the underlying forecast values at each
-                    percentile.
+            Array of percentiles from a Cumulative Distribution Function.
+        forecast_at_percentiles (Numpy array):
+            Array containing the underlying forecast values at each
+            percentile.
+        bounds_pairing (Tuple):
+            Lower and upper bound to be used as the ends of the
+            cumulative distribution function.
```

Comment:
This docstring change looks unintentional - please would you revert this.

Reply:
Good spot. Done.